modules/git/commit.go

```diff
@@ -253,15 +253,15 @@ func NewSearchCommitsOptions(searchString string, forAllRefs bool) SearchCommits
 	for _, k := range fields {
 		switch {
 		case strings.HasPrefix(k, "author:"):
-			authors = append(authors, strings.TrimPrefix(k, "author:"))
+			authors = append(authors, strings.ReplaceAll(strings.TrimSpace(strings.TrimPrefix(k, "author:")), "'", ""))
 		case strings.HasPrefix(k, "committer:"):
-			committers = append(committers, strings.TrimPrefix(k, "committer:"))
+			committers = append(committers, strings.ReplaceAll(strings.TrimSpace(strings.TrimPrefix(k, "committer:")), "'", ""))
 		case strings.HasPrefix(k, "after:"):
-			after = strings.TrimPrefix(k, "after:")
+			after = strings.ReplaceAll(strings.TrimSpace(strings.TrimPrefix(k, "after:")), "'", "")
 		case strings.HasPrefix(k, "before:"):
-			before = strings.TrimPrefix(k, "before:")
+			before = strings.ReplaceAll(strings.TrimSpace(strings.TrimPrefix(k, "before:")), "'", "")
 		default:
-			keywords = append(keywords, k)
+			keywords = append(keywords, strings.ReplaceAll(strings.TrimSpace(k), "'", ""))
```

Comment:
I don't see the necessary of replacing quotes here. Each of these arguments will be added by `"--opt="+v` as a single CLI argument and will be escaped by exec.Cmd. 

Otherwise, the problem is, why not other characters like space, double-quote, zero-endding, new-line got processed here? They do not need to either IMO.

Reply:
well hope so